opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -90,18 +95,20 @@ public TracingFilter() {
      * @param tracer
      */
     public TracingFilter(Tracer tracer) {
-        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS));
+        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS), null);
     }
 
     /**
      *
      * @param tracer tracer
      * @param spanDecorators decorators
+     * @param skipPattern null or pattern to exclude certain paths from tracing e.g. "/health"
      */
-    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators) {
+    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators, Pattern skipPattern) {
```

Comment:
> For backward compatibility with rules, need to maintain the existing two parameter constructor, so can you add a third constructor.

Rules are versioned right? so could you update rules when upgrading version of this component? This is still `0.0.x` I would like to keep it clean and care about these kind of things once it is `1.x`.

> It might also be worth supporting a system property/env var to define the pattern, for example, when used with the javaagent. 

I guess javaagent can read system/env props and set it when installing it. If it would be useful for more people I guess we can support it.